src/app/private/scripts/controllers/collectionPlayerController.js

```diff
@@ -171,6 +176,31 @@
           $state.go('Resources')
         }
       }
+
+      cpvm.generateStartEvent = function () {
```

Comment:
This is repeat method and code with different params and data move it to service